app/assets/javascripts/components/generic_object/main-custom-button-form.js

```diff
@@ -185,14 +185,16 @@ function mainCustomButtonFormController(API, miqService, $q, $http) {
       display_for: vm.customButtonModel.display_for,
       submit_how: vm.customButtonModel.submit_how,
     };
-
+debugger;
```

Comment:
This needs to go away :smile:

Reply:
I missed the little (de)bugger 😄 Fixed :)